services/server/src/sourcify-chains-default.json

```diff
@@ -1892,5 +1892,21 @@
     "sourcifyName": "TixChain Testnet",
     "supported": true,
     "rpc": ["https://subnets.avax.network/tixchain/testnet/rpc"]
+  },
+  "223": {
+    "sourcifyName": "B2 Mainnet",
+    "supported": true,
+    "etherscanApi": {
+      "apiURL": "https://explorer.bsquared.network/api"
```

Comment:
Is this also supporting the `/api?module=contract&action=getcontractcreation`? I cannot find it in the documentation. Would we need an API key for using it?